### PR TITLE
Changing VMware.WorkstationPro version 16.1.1

### PR DESCRIPTION
**Pull request by ImJoakim**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/18334)

### CODE REVIEW

File: manifests/v/VMware/WorkstationPro/16.1.1/VMware.WorkstationPro.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+PackageIdentifier: VMware.WorkstationPro
+PackageVersion: 16.1.1
+PackageLocale: en-US
+Publisher: VMware, Inc.
+PackageName: VMware Workstation
+PackageUrl: https://www.vmware.com/products/workstation-pro.html
+License: (c) 2021 VMware, Inc. All rights reserved.
+LicenseUrl: https://www.vmware.com/help/legal.html
+ShortDescription: VMware Workstation Pro is the industry standard for running multiple operating systems as virtual machines (VMs) on a single Linux or Windows PC to build, test, or demo software.
+Moniker: vmware-workstation
```
Comment (KaranKad) [inline manifests/v/VMware/WorkstationPro/16.1.1/VMware.WorkstationPro.locale.en-US.yaml:13]:
```suggestion
Moniker: vmware-workstation-pro
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Comment by KaranKad** [comment]
Looks like validation has been stopped for a while.
Also winget isn't working 
image

**Comment by denelon** [comment]
@KaranKad this was due to an upgrade of the pipelines, and a rebuild of the PreIndexed package. We're looking at a way to eliminate the temporary outage caused when this happens. It was part of publishing the duplicate hash check feature. We've got one more PR to merge on the back end, and then we will enable the feature. You should see "Possible-Duplicate" tag added to PRs when we detect a duplicate "InstallerSha256" one this is fully implemented.

**Comment by ImJoakim** [comment]
@denelon Any reason why some of my PRs are stuck on the Installer check? Some of them went through, but these appear to be stuck. I tried rerunning them myself by closing and opening the PR, but it still gets stuck.

#18327 
#18332 
#18334
#18425
#18421

**Comment by ImJoakim** [comment]
Seems like the 5 PRs that I mentioned above are still stuck, I'll try to close them and open new ones.

**Pull request closed without merging**